src/libraries/System.IO.FileSystem.Watcher/src/System/IO/FileSystemWatcher.Linux.cs

```diff
@@ -514,6 +514,9 @@ private void CancellationCallback()
             /// </summary>
             private void ProcessEvents()
             {
+                // It's OK to set the name since we specified TaskCreationOptions.LongRunning
```

Comment:
Is there a good reason to use `TaskCreationOptions.LongRunning` here instead of just creating the thread directly? `TaskCreationOptions.LongRunning` just adds some extra overhead vs. creating the thread directly.

Reply:
> Is there a good reason

Today, no. I believe it was written this way when Thread wasn't exposed yet.